test-requirements.txt

```diff
@@ -24,7 +24,7 @@ pytest-reorder~=0.1.0
 git+https://github.com/jayvdb/pytest-reqs@coala#egg=pytest-reqs
 pytest-timeout~=1.3.0
 pytest-travis-fold~=1.3.0
-pytest-xdist~=1.15
+pytest-xdist==1.15
```

Comment:
This needs a separate commit/PR.

Reply:
I’ve added a separate commit for just this change. Will it be sufficient? Or Should i create a new PR for that ?

Reply:
It's an entirely different change, so I'd vote for a new PR.

Reply:
Will create a new PR for it 👍

Reply:
This has been fixed in [03ae7df](https://github.com/coala/coala/commit/03ae7df13980cf08b9f84083ab1ac7c19fe839c8). Can you remove this change and push again?